frontend/packages/vsphere-plugin/src/components/VSphereConnectionForm.css

```diff
@@ -0,0 +1,7 @@
+.vsphere-connection-form-helper__datacenter {
```

Comment:
lets align the css rules naming.
In previous css you're using a different prefix
```
.plugin-vsphere-status-popup
```

Reply:
done